TERRA_QUICKSTART.md

```diff
@@ -30,6 +30,10 @@ $ ifconfig | grep inet | grep broadcast
     inet 192.XXX.Y.ZZZ netmask 0xffffff00 broadcast 192.XXX.Y.255
 $ export CROMWELL_URL="http://192.XXX.Y.ZZZ:8000/api/workflows/v1"
 ```
+or the one-liner:
+```
+$ export CROMWELL_URL="http://$(ifconfig | grep inet | grep broadcast | cut -d ' ' -f 2):8000/api/workflows/v1"
```

Comment:
NICE

Reply:
I've gotten this wrong more than once before by either copying the broadcast address instead of the computer's address or forgetting to include the port number, so having the one-liner helps me avoid those mistakes.